src/stampy_nlp/settings.py

```diff
@@ -0,0 +1,30 @@
+import os
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+CODA_TOKEN: str = os.getenv('CODA_TOKEN')
+HUGGINGFACE_API_KEY: str = os.getenv('HUGGINGFACE_API_KEY')
+PINECONE_API_KEY: str = os.getenv('PINECONE_API_KEY')
+
+STAMPY_BUCKET: str = os.getenv('STAMPY_BUCKET', 'stampy-nlp-resources')
+DUPLICATES_FILENAME: str = os.getenv('DUPLICATES_FILENAME', 'stampy-duplicates.json')
+AUTH_PASSWORD: str = os.getenv('AUTH_PASSWORD', 'miko')
```

Comment:
this will have to be set for the google cloud run workers and in github secrets. I don't have access, which is good :D

Reply:
LOL! OK, I've added AUTH_PASSWORD to the GH repo & GCP secrets, so if you fix the placeholder passwords, we should be good to go!